.github/workflows/pr-enhanced-labeler.yml

```
@@ -0,0 +1,341 @@
+            // Get file paths
+            const paths = files.map(f => f.filename);
+
+            core.setOutput('total_changes', totalChanges);
+            core.setOutput('files_changed', filesChanged);
+            core.setOutput('paths', JSON.stringify(paths));
+
+            return { totalChanges, filesChanged, paths };
+
+      - name: Determine size label
+        id: size-label
+        run: |
```

actionlint (1.7.9)

property "size-label" is not defined in object type {pr-files: {conclusion: string; outcome: string; outputs: object}}

(expression)